boards/arm/nrf51_pca10028/nrf51_pca10028.yaml

```diff
@@ -12,6 +12,7 @@ supported:
   - ble
   - nvs
   - pwm
+  - spi
```

Comment:
Do you need to update the board doc too?

Reply:
No, SPI is already listed as supported feature in documentation of both boards modified by this PR.